### PR TITLE
Globalization landing page

### PR DESCRIPTION
**Pull request by NilsJacobsen**
Hey folks, 

can I get feedback on the new wording?

**What changed?**
- H1
- Subtitle of H1
- Card text

**Most current version:**

localhost_3000_ (3)

> Let use the thumbs to see if you like it? Gonna update the screenshot if we change something.

### CODE REVIEW

**Comment by NilsJacobsen** [comment]
@inlang/team feedback welcome

**Comment by samuelstroschein** [comment]
Hmm, even though I suggest "infrastructure to go global", reading it on the landing page seems less catchy because it misses that we target apps/software at the moment. 

---

### Infrastructure to take your app global [Alternative 1] 

Inlang's ecosystem makes adapting your application to different markets easy.

---

### Globalization infrastructure for software [Alternative 2] 

Inlang's ecosystem makes adapting your application to different markets easy.

---

I think I vote for [Alternative 2] because app sounds too narrow while "for software" still catches app developers but a large group gets hooked. 

What do you guys think?

**Comment by felixhaeberle** [comment]
Great additions here 😙

Some feedback points which could be implemented:

- header image: CI: inlang validate -> CLI: npx @inlang/cli lint 👍 we currently do not have CI ootb, we soon can state Github Action here, but for now, CLI instead of CI would be better. andfurthermore, giving something what can actually be executed and doesn't feel like a "dummy". 
- button "Go global now". A bit too much. "Get started" is much better / lower friction clicking it. 
- Alternative 2 & addition to subheadline:
Inlang's ecosystem makes adapting your application to different markets easy. *Start today with our i18n apps.* (I added the italic part, should not be italic ofc).
- flex with cal.com and other popular repositories which are big and are known - kinda revive the logo section we already had but now only with known bangers?
- app cards are **a bit** too clean overall, it could be more engaging. don't know why I feel like that and it's only a minor, but want to share my impression / feeling of looking at those cards.
- button "plugins" could be more engaging, but don't know how. overall this space could benefit from improvements. maybe add i18next (any the others) here somewhere? that you get it immediately that users can use inlang with this library RIGHT NOW. if I'm a i^8next user already, I will be hooked by this. and it's good for seo to have i18next somehwere when they for example google i18next ide extension -> inlang
- `ideExtensionPlugin()` does not exist anymore and can be deleted from the config example
- have an information / banner (light grey something) on `npx @inlang/cli config init` right under the config example to easily know how this can be automatically done for you & provide an CTA to go to documentation for manual creation process.
- remove "read discussion on github" link on the pricing section. I don't think people are interested rn.
- add potential last section which again states to "Get Started" and to jump into docs or give potential starting points like:
1. Create inlang.config.js now -> `npx @inlang/cli config init`
OR
2. Explore our supported libraries (i18next, typesafe-i18n, ...) -> plugin registry
OR
3. Get started with starter template  [1min] -> inlang/example


As always, take what you deem right. 🤗

**Comment by NilsJacobsen** [comment]
Thanks for the overall feedback guys. 

Maybe at first I think this discussion should only be about the changes. 
I would suggest open new issues for thinks that feel off. I will react on the stuff anyway:

### Samuels title changes: 

I feel like `Globalization infrastructure for software` is a bit bulky. It is just not super easy to read because the words are so long. I agree that it should be pointing towards software. Another idea: "Taking your software worldwide with globalization infrastructure."

Maybe the subtitle: "Inlang's ecosystem makes adapting your application to different markets easy." Could be improved because it is not really telling what makes it outstanding. And makes it easy is nowadays something that everybody wants to do.

### Felix recommendations:

#### Header

I agree that the header image could be improved but i would suggest creating an other PR for that. There we can discuss what needs to be changed. 

I changed the butten text to be different to the other one in the next section. But I agree could be a bit more calm. We can change it to `Get started` again if everybody agrees.

We can do the logos on the page when bigger repos are using us (cal.com is using us):D We can do this in an other improvement.

#### Ecosystem

I know but I feel like it is better than before. For now i think this is fine.
When I designed the plugin surface it shouldn't be a button :DDD That was probably the problem. I don't know yet. I guess at some point we again refactoring this section. But I think for some weeks that could still be fine. We could put the ide extension in the example config.

#### Config
@felixhaeberle could you create an issue for the init command. So we can handle that elsewhere. There's so many topic going on and I think that is a great idea.

I'm gonna remove the ideExtension ...

#### Pricing
I could remove it for now if everybody agrees.

#### I think the new section is not necessarily part of this discussion. Maybe open a separate issue for that.

---

What I directly take away:
- Remove ide extension from config
- add i18next plugin instead of jsonPlugin in config
- Remove button on pricing section

**Comment by NilsJacobsen** [comment]
### While looking at that i'm not to upset:D

image

**Comment by jannesblobel** [comment]
I agree with your takeaway ( btw nice feedback, nothing to add)
except for :
> Remove button on pricing section

I would rephrase it into ` read more` or `more information`

**Comment by felixhaeberle** [comment]
@NilsJacobsen Thank you for answering my feedback in detail. 

Your suggestion of creating new issues for it seems reasonable because we don't want huge issues which get out of control.

I think doing better on the plugin communication (support of i18next and others) & give a last cta section which catches different personas / intentions could massively improve growth because targeting a broader audience and **give easy actionables** for those people.

I will move on and create issues for the improvements which are not in this scope. @floriandwt Do you also want to touch and improve the marketing page?

**Comment by samuelstroschein** [comment]
@NilsJacobsen 

- I vote for globalization infrastructure for software. 
- Please don't remove the pricing section but feel free to remove "read github discussion"

Off-topic: Giving feedback on website copy is uff hard with linear messages. Looking forward to dogfood inlang and build something that makes this easier!

**Comment by NilsJacobsen** [comment]
Okey let's bring the discussion again to only the headline and subheadline:
So i guess we agreed on "Globalization infrastructure for software". I think only with the marked globalization it is relatively easy to ready (To answer @felixhaeberle  question there are huge words in it and i caught myself reading it super slow). But I think like on the image it seems to be okey. Please react with an thumbs up on this message if you agree.

**Comment by NilsJacobsen** [comment]
So now let's talk about the subheadline. @samuelstroschein proposed:
"Inlang's ecosystem makes adapting your application to different markets easy." I reacted with: "It is not really telling what makes it outstanding. And makes it easy is nowadays something that everybody wants to do." Why should ask us why people use us already and why we compete against legacy products. I really liked the end of my proposal: "... to collaborate seamlessly using a single source of information that is version controlled." also "for devs" is outstanding, "setup in a minute" is also something that is pretty cool. What do you think?

**Comment by floriandwt** [comment]
> @floriandwt Do you also want to touch on and improve the marketing page?

Yes, marketing page = landing page I guess? :D

Generally, I feel like "Globalization infrastructure for software" is a good and simple explanation. However, globalization is a word you don't hear quite often on such websites – could be beneficial for us (standing out, especially when a business person is visiting) or not (because people could get overwhelmed by the big word, especially devs).

If we go with that title, "Inlang's ecosystem makes adapting your application to different markets easy." would be a good subheadline because it explains the whole topic from the same standpoint as the headline.

Possible to do two separate versions with different wording (one focusing on globalization and one focusing on the development aspect)? Like an A/B testing.

**Comment by NilsJacobsen** [comment]
I feel like: "Inlang's ecosystem makes adapting your application to different markets easy." That could be the subtitle of every competitor that we have. Where is the uniqueness of our product? Why should they even care about inlang?

**Comment by NilsJacobsen** [comment]
I feel like we already tell this with infrastructure. But if you all want it you overrule me. Please vote on this comment with thumbs. 
-> "Inlang's ecosystem makes adapting your application to different markets easy."

**Comment by NilsJacobsen** [comment]
okey **no fan** but i'm gonna add it

**Comment by jannesblobel** [comment]
I know this is not a big deal, but I would change the colors of the word `Globalization.` It looks better and focuses on the beginning of the word instead of the end. 
CleanShot 2023-06-22 at 20 39 27@2x
CleanShot 2023-06-22 at 20 40 35@2x

**Comment by felixhaeberle** [comment]
Agree 👍

**Comment by samuelstroschein** [comment]
I would take away the gradient :)

**Comment by NiklasBuchfink** [comment]
It has already been changed to what @jannesblobel suggested, am I right?

**Comment by jannesblobel** [comment]
@samuelstroschein like this ?
CleanShot 2023-06-23 at 10 02 40@2x

**Comment by NiklasBuchfink** [comment]
We can use the gradient without transparency if that is the problem.

**Comment by samuelstroschein** [comment]
@jannesblobel just no color. But tbh this is minor. I take myself out of the discussion/we shouldn't spend too much time on this

**Comment by jannesblobel** [comment]
I suggest not putting any more time into the landig page and just let it be for now.
